### PR TITLE
Added test that tests snapshot migration from manager 3.2.1 to manager

### PR DESCRIPTION
**Pull request by xwombat**
3.3

### CODE REVIEW

File: cosmo_tester/resources/blueprints/snapshots-migration-321-to-33/openstack-manager-blueprint.yaml [hunk 1]
```diff
@@ -0,0 +1,493 @@
+tosca_definitions_version: cloudify_dsl_1_0
```
Comment (simon-bar) [inline cosmo_tester/resources/blueprints/snapshots-migration-321-to-33/openstack-manager-blueprint.yaml]: What would be better is to wget this file and patch it like Pawel has done in his test:

https://github.com/codilime/cloudify-system-tests/blob/efa0f9eb886679f2c095eaa4b59dee2dc070a40b/cosmo_tester/test_suites/test_snapshots/snapshots_multiple_managers_test.py#L68

File: cosmo_tester/resources/scripts/snapshots_migration_scripts/bootstrap_manager_3_2_1.template [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+cd {{ work_dir }}
+source {{ venv_name }}/bin/activate >/dev/null
+
+cd {{ repo_path }}
+git checkout 3.2.1-build
+cp {{ work_dir }}/{{ updated_blueprint }} ./openstack/
+cd {{ work_dir }}/{{ venv_name }}
+cfy bootstrap --install-plugins -i {{ work_dir }}/{{ inputs_file }} -p {{ repo_path }}/openstack/{{ updated_blueprint }}
+
+sleep 10
```
Comment (simon-bar) [inline cosmo_tester/resources/scripts/snapshots_migration_scripts/bootstrap_manager_3_2_1.template]: Why sleep here? If manager is not "really ready" straight after bootstrap, I would perform some sleep-loop until, let's say, cfy status returns good news or I don't know, cfy deployments list exits with 0..

File: cosmo_tester/resources/scripts/snapshots_migration_scripts/clear_manager_3_2_1.template [hunk 3]
```diff
@@ -0,0 +1,13 @@
+set -eax
+orig_dir=$(pwd)
+cd {{ work_dir }}
+virtualenv {{ venv_name }} >/dev/null
```
Comment (simon-bar) [inline cosmo_tester/resources/scripts/snapshots_migration_scripts/clear_manager_3_2_1.template]: Why /dev/null here? :)

File: cosmo_tester/resources/scripts/snapshots_migration_scripts/create_and_download_snapshot_3_2_1.template [hunk 4]
```diff
@@ -0,0 +1,14 @@
+set -eax
+orig_dir=$(pwd)
```
Comment (simon-bar) [inline cosmo_tester/resources/scripts/snapshots_migration_scripts/create_and_download_snapshot_3_2_1.template]: I'm not sure if those orig_dirs are necessary if you use subprocess

File: cosmo_tester/test_suites/test_snapshots_from_321_to_33/hello_world_old_new_manager_test.py [hunk 5]
```diff
@@ -0,0 +1,555 @@
...
+OLD_MANAGER_TEMPLATE_NAME = 'manager_3_2_1_inputs.template'
+OLD_MANAGER_INPUTS_NAME = 'manager_3_2_1_inputs.yaml'
+
+NEW_MANAGER_TEMPLATE_NAME = 'manager_3_3_inputs.template'
+NEW_MANAGER_INPUTS_NAME = 'manager_3_3_inputs.yaml'
+
+UPDATE_SCRIPT_NAME = 'update_user_and_credentials_3_2_1.py'
+
+
+class HelloWorldOldNewManagerTest(TestCase):
```
Comment (simon-bar) [inline cosmo_tester/test_suites/test_snapshots_from_321_to_33/hello_world_old_new_manager_test.py]: Maybe HelloWorldSnapshotMigrationFrom3_2_1to3_3Test
or
Maybe HelloWorldSnapshotMigrationFrom_3_2_1_to_3_3_Test

I know.. it's hard to come up with a good name here :/

File: cosmo_tester/resources/blueprints/snapshots-migration-321-to-33/openstack-manager-blueprint.yaml (hunk 1 above)
Comment (idanmo) [inline cosmo_tester/resources/blueprints/snapshots-migration-321-to-33/openstack-manager-blueprint.yaml]: Yes just download the relevant manager blueprint version.

**Pull request merged**